aiowebostv/webos_client.py

```diff
@@ -307,62 +316,50 @@ async def connect_handler(self, res: Future) -> None:
     async def callback_handler(
         queue: Queue[dict[str, Any]],
         callback: Callable,
-        future: Future[dict[str, Any]] | None,
+        future: Future[dict[str, Any]],
     ) -> None:
         """Handle callbacks."""
         with suppress(asyncio.CancelledError):
             while True:
                 msg = await queue.get()
                 payload = msg.get("payload")
                 await callback(payload)
-                if future is not None and not future.done():
+                if not future.done():
                     future.set_result(msg)
 
-    async def consumer_handler(
-        self,
-        web_socket: ClientWebSocketResponse,
-        callbacks: dict[int, Callable] | None,
-        futures: dict[int, Future] | None,
-    ) -> None:
-        """Callbacks consumer handler."""
-        callback_queues: dict[int, Queue[dict[str, Any]]] = {}
-        callback_tasks: dict[int, Task] = {}
+    async def _process_text_message(self, data: str) -> None:
+        """Process text message."""
+        if not self.callbacks and not self.futures:
+            return
```

Comment:
I wonder if this can be removed as besides fetching the message id we do nothing if they don't exist